lib/api/src/sys/module.rs

```diff
@@ -279,10 +279,13 @@ impl Module {
         resolver: &dyn Resolver,
     ) -> Result<InstanceHandle, InstantiationError> {
         unsafe {
+            let path = self.store.path();
```

Comment:
this looks like it does not need to be inside the unsafe block, if so, move it outside.